iosApp/iosApp.xcodeproj/project.pbxproj

```diff
@@ -351,7 +351,7 @@
 				CODE_SIGN_IDENTITY = "Apple Development";
 				CODE_SIGN_STYLE = Automatic;
 				DEVELOPMENT_ASSET_PATHS = "\"iosApp/Preview Content\"";
-				DEVELOPMENT_TEAM = "${TEAM_ID}";
```

Comment:
@ljubicics Maybe we need to remove this file from git since you added it to .gitignore?

Reply:
After some research, found out that `project.pbxproj` contains configuration properties for Xcode. Given so, I removed it from `.gitignore` and rollbacked the file to original state.